src/util/projectDelete.tsx

```diff
@@ -0,0 +1,54 @@
+import { ReactNode } from "react";
+import { filterUsedByType } from "./usedBy";
+import { ResourceType } from "./resourceDetails";
+import { LxdProject } from "types/project";
+import { Link } from "react-router-dom";
+
+export const generateProjectUsedByTooltip = (project: LxdProject) => {
```

Comment:
This seems pretty tailored towards the `DeleteProjectBtn`, shouldn't the function live in that component file?

Reply:
good point, I've moved the function into `DeleteProjectBtn` file now.